tests/net/ipv6_fragment/testcase.yaml

```diff
@@ -1,8 +1,13 @@
 common:
```

Comment:
nit; Typo in the commit message: `tests: net: ipvr6_fragment:` -> `tests: net: ipv6_fragment:`